README.md

```diff
@@ -1,7 +1,49 @@
 # hasura-bot :robot:
 
-Hasura's own GitHub bot.
+Hasura's own GitHub bot, currently tasked with commenting on pull requests when they are opened/closed/merged.  
+Deployed on [Zeit Now](https://zeit.co/now): https://hasura-github-bot.now.sh
 
-Currently tasked with commenting on pull requests when they are opened/closed/merged.
+## Local Setup
+First, you will need to create an account on [vercel](https://vercel.com/signup).
 
-Deployed on [Zeit Now](https://zeit.co/now): https://hasura-github-bot.now.sh
+Next, set up `ngrok` by following the instructions given [here](https://ngrok.com/download).
```

Comment:
Probably remove this line from here and mention the download link in the "Set up port forwarding with `ngrok`" step instead.